packages/client/modules/email/components/LimitsEmails/LockedEmail.tsx

```diff
@@ -0,0 +1,78 @@
+import {ContactInfo} from 'parabol-client/types/constEnums'
+import React from 'react'
+import {EMAIL_CORS_OPTIONS} from '../../../../types/cors'
+import makeAppURL from '../../../../utils/makeAppURL'
+import {emailCopyStyle, emailLinkStyle} from '../../styles'
+import Button from './../Button'
+import EmailBlock from './../EmailBlock/EmailBlock'
+import EmailFooter from './../EmailFooter/EmailFooter'
+import EmptySpace from './../EmptySpace/EmptySpace'
+import Header from './../Header/Header'
+import Layout from './../Layout/Layout'
+
+const innerMaxWidth = 480
+
+const copyStyle = {
+  ...emailCopyStyle
+}
+
+const linkStyle = {
+  ...emailCopyStyle,
+  ...emailLinkStyle
+}
+
+export interface LimitsEmailProps {
+  appOrigin: string
+  orgId: string
+  preferredName: string
+  orgName: string
+}
+
+export default function LockedEmail(props: LimitsEmailProps) {
+  const {appOrigin, preferredName, orgId, orgName} = props
+  const billingURL = makeAppURL(appOrigin, `/me/organizations/${orgId}/billing`, {
+    searchParams: {
+      utm_source: 'notification email',
+      utm_medium: 'email',
+      utm_campaign: 'locked-email'
+    }
+  })
+  return (
+    <Layout maxWidth={544}>
+      <EmailBlock innerMaxWidth={innerMaxWidth}>
+        <Header align='left' appOrigin={appOrigin} corsOptions={EMAIL_CORS_OPTIONS} />
+        <p style={{...copyStyle, marginBottom: '0px'}}>{`Hi ${preferredName} 👋`}</p>
+        <EmptySpace height={16} />
+        <p style={{...copyStyle, marginBottom: '0px'}}>
+          {'Unfortunately, '}
+          <span style={{fontWeight: 600}}>
+            {`${orgName} has exceeded the two-team limit on the Starter Plan for more than 30 days, and your account has been deactivated.`}
```

Comment:
Is it ok to hardcode numbers or should we take it from Threshold?

Reply:
Thanks - updated!